README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://img.shields.io/gitlab/pipeline/zach_nation/coremltools/master)](https://gitlab.com/zach_nation/coremltools/-/pipelines?page=1&scope=branches&ref=master)
+[![Build Status](https://img.shields.io/gitlab/pipeline/zach_nation/coremltools/main)](https://gitlab.com/zach_nation/coremltools/-/pipelines?page=1&scope=branches&ref=main)
```

Comment:
You're right it should be `main` not `master`.

We've also moved away for the "zach_nation" GitLab repository. We have set up a team repository with the username "coremltools1".

Please also change the two occurrences of "zach_nation" to "coremltools1".